README.md

```diff
@@ -117,21 +118,21 @@ System Properties > Advanced > Environment Variables
 
 Copy the value of NVM_HOME in the admin user to the value of NVM_HOME in the System variables
 
-  After installing NVM, global utilities (e.g. yarn) will have to be reinstalled for each installed version of node:
+After installing NVM, global utilities (e.g. yarn) will have to be reinstalled for each installed version of node:
 
-  ```bash
-    nvm use 16.0.0
-    npm install -g yarn
-    nvm use 14.16.1
-    npm install -g yarn
-    ```
+````bash
```

Comment:
This opening and it's corresponding closing backticks are unnecessary are cause the formatting to be incorrect:
![image](https://user-images.githubusercontent.com/1875867/231732796-a5c9bc95-46d0-4581-803c-620737a85ab8.png)

They should be removed.
```suggestion
```bash
```